napari_btrack/_tests/test_dock_widget.py

```diff
@@ -32,79 +33,92 @@ def test_add_widget(make_napari_viewer):
         widget_name="Track",
     )
 
-    assert len(list(viewer.window._dock_widgets)) == num_dw + 1  # noqa: S101
+    assert len(list(viewer.window._dock_widgets)) == num_dw + 1
 
 
 @pytest.fixture
 def track_widget(make_napari_viewer) -> Container:
     """Provides an instance of the track widget to test"""
     make_napari_viewer()  # make sure there is a viewer available
-    return track()
+    return napari_btrack.main.create_btrack_widget()
 
 
 @pytest.mark.parametrize("config", [cell_config(), particle_config()])
 def test_config_to_widgets_round_trip(track_widget, config):
     """Tests that going back and forth between
     config objects and widgets works as expected.
     """
-    expected_config = load_config(config)
-    _update_widgets_from_config(track_widget, expected_config)
-    actual_config = _widgets_to_tracker_config(track_widget)
-    # use json.loads to avoid failure in string comparison because e.g "100.0" != "100"
-    assert json.loads(actual_config.json()) == json.loads(  # noqa: S101
-        expected_config.json()
-    )
 
+    expected_config = btrack.config.load_config(config).json()
 
-@pytest.fixture
-def user_config_path() -> str:
-    """Provides a (dummy) string to represent a user-provided config path."""
-    return "user_config.json"
+    unscaled_config = napari_btrack.config.UnscaledTrackerConfig(config)
+    napari_btrack.sync.update_widgets_from_config(unscaled_config, track_widget)
+    napari_btrack.sync.update_config_from_widgets(unscaled_config, track_widget)
+
+    actual_config = unscaled_config.scale_config().json()
 
+    # use json.loads to avoid failure in string comparison because e.g "100.0" != "100"
+    assert json.loads(actual_config) == json.loads(expected_config)
```

Comment:
I'm not sure...let me check

Reply:
ah, I removed the `user_config_path` fixture as it's now only being used in a single test (one that tests saving files). Previously it was also used in a test for loading files and be used to mock a loaded config. Now, the test for loading files loads an actual config rather than mocking one